tests/test_visitors/test_ast/test_builtins/test_numbers/test_approximate_constants.py

```diff
@@ -24,10 +26,15 @@ def test_violation_on_approximate_constants(
     parse_ast_tree,
     default_options,
     variable_value,
+    parse_tokens,
 ):
     """Ensures that usage of approximate constants not allowed."""
-    tree = parse_ast_tree(f'my_const = {variable_value}')
-    visitor = WrongNumberVisitor(default_options, tree=tree)
+    variable_str = f'my_const = {variable_value}'
+    tree = parse_ast_tree(variable_str)
+    file_tokens = parse_tokens(variable_str)
+    visitor = WrongNumberVisitor(
+        default_options, tree=tree, file_tokens=file_tokens
+    )
```

Comment:
Please, add some extra `variable_value` examples here: we need to test that the formatting is preserved.

For example: `3.1_415` will be formatted exactly like `3.1_415` and not like `3.1415`